site/styles/_footer.scss

```diff
@@ -52,16 +57,17 @@ footer {
 
 .footer-social-media-icons {
   display: grid;
-  gap: 24px;
-  grid-auto-columns: 25px;
+  gap: 16px;
+  grid-auto-columns: 32px;
   grid-auto-flow: column;
   grid-template-rows: 1fr;
   justify-content: center;
-  margin-bottom: 30px;
+  margin-left: auto;
+  padding: 0 16px 16px;
```

Comment:
Is this padding necessary?  There is already a big margin between the social buttons and the footer info bar on mobile!

Reply:
It was just added to keep consistency with the existing nav elements. Honestly looks better without it though